src/components/ebay-progress-bar/index.marko

```diff
@@ -0,0 +1,7 @@
+import processHtmlAttributes from "../../common/html-attributes"
+
+static var ignoredAttributes = ["class"];
+
+<progress
+    ...processHtmlAttributes(input, ignoredAttributes)
```

Comment:
Can we default `min` and `max` to 0 and 100?
You can do something like:
`min = (input.min || 0)`
`max = (input.max || 100)`
And add that info in the readme and update the examples, but keep 1 with a custom min/max